test/parallel_api/ranges/equal_ranges_sycl.pass.cpp

```diff
@@ -0,0 +1,57 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include <oneapi/dpl/execution>
+
+#include "support/pstl_test_config.h"
+
+#if _ENABLE_RANGES_TESTING
+#include <oneapi/dpl/ranges>
+#endif
+
+#include "support/utils.h"
+
+#include <iostream>
+
+int32_t
+main()
+{
+#if _ENABLE_RANGES_TESTING
+    constexpr int max_n = 10;
+    int data1[max_n]     = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
+    int data2[max_n]     = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
+    int data3[max_n]     = {0, 1, 2, -1, 4, 5, 6, 7, 8, 9};
+
+    bool res1 = false, res2 = false;
+    using namespace oneapi::dpl::experimental::ranges;
+    {
+        sycl::buffer<int> A(data1, sycl::range<1>(max_n));
+        sycl::buffer<int> B(data2, sycl::range<1>(max_n));
+        sycl::buffer<int> C(data3, sycl::range<1>(max_n));
+
+        auto view = views::all(A);
+                                       
+        res1 = equal(TestUtils::default_dpcpp_policy, view, B);
```

Comment:
Please rebind the policy to avoid collisions of kernel names: here and in the other tests.